src/libraries/System.Private.CoreLib/src/System/Runtime/Intrinsics/ISimdVector_2.cs

```diff
@@ -0,0 +1,562 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System.Numerics;
+using System.Runtime.CompilerServices;
+using System.Runtime.InteropServices;
+
+namespace System.Runtime.Intrinsics
+{
+    /// <summary>Defines a single instruction, multiple data (SIMD) vector type.</summary>
+    /// <typeparam name="TSelf">The type that implements the interface.</typeparam>
+    /// <typeparam name="T">The type of the elements in the vector.</typeparam>
+    internal unsafe interface ISimdVector<TSelf, T>
+        : IAdditionOperators<TSelf, TSelf, TSelf>,
+       // IAdditiveIdentity<TSelf, TSelf>,
```

Comment:
Why are all of these interfaces commented out? Is that just because they've not been needed yet for the APIs that have implemented these and thus haven't been done yet?  Or are they not relevant?  A comment would be helpful.

Reply:
Ones that aren't needed or that aren't trivial to support yet.

Still wanting to cover them in as part of a follow up, but not crucial to getting the baseline support in so we can start using everything internally.

I can add a comment.